internal/bminventory/inventory.go

```diff
@@ -1621,11 +1622,15 @@ func (b *bareMetalInventory) generateClusterInstallConfig(ctx context.Context, c
 		return errors.Wrapf(err, "failed to get install config for cluster %s", cluster.ID)
 	}
 
-	releaseImage, err := b.versionsHandler.GetReleaseImage(cluster.OpenshiftVersion)
+	releaseImage, err := b.versionsHandler.GetReleaseImage(cluster.OpenshiftVersion, versions.DefaultCPUArchitecture)
+	if err != nil {
+		msg := fmt.Sprintf("failed to get release image for cluster %s with openshift version %s", cluster.ID, cluster.OpenshiftVersion)
+		return errors.Wrapf(err, msg)
+	}
 
 	if err != nil {
```

Comment:
Isn't it a dead code? you added `if err != nil {` with return few lines above

Reply:
right, some leftovers in rebase, need to remove lines 1631-1634.